engines/bastion_katello/app/assets/javascripts/bastion_katello/environments/content.service.js

```diff
@@ -87,11 +88,14 @@
         };
 
         this.buildNutupane = function (params) {
-            var nutupane;
+            var nutupane, type, disableAutoLoad;
+
+            type = this.getCurrentContentType();
+            disableAutoLoad = type.autoLoad !== true;
```

Comment:
Any reason to not just use `!type.autoload` below?

Reply:
though I can see how assigning it to a variable is more readable and explicit

Reply:
Good suggestion - I like it better! Updated